SofaKernel/modules/SofaHelper/src/sofa/helper/ComponentChange.h

```diff
@@ -102,8 +102,8 @@ class SOFA_HELPER_API Moved : public ComponentChange
 public:
     Moved(std::string  sinceVersion, std::string fromPlugin, std::string toPlugin) {
         std::stringstream output;
-        output << "This component has been moved from " << fromPlugin << " to " << toPlugin << " since SOFA " << sinceVersion << "."
-               <<  "To continue using this component you need to update your scene "
+        output << "This component has moved from " << fromPlugin << " to " << toPlugin << " since SOFA " << sinceVersion << "."
+               <<  "To continue using this component you may need to update you scene "
```

Comment:
```suggestion
        output << "This component has been MOVED from " << fromPlugin << " to " << toPlugin << " since SOFA " << sinceVersion << ". "
               <<  "To continue using this component you may need to update your scene "
```